internal/provider/resource_vault_cluster.go

```diff
@@ -65,6 +66,14 @@ func resourceVaultCluster() *schema.Resource {
 				ValidateDiagFunc: validateSlugID,
 			},
 			// Optional fields
+			"project_id": {
+				Description:  "The ID of the HCP project where the Vault cluster is located.",
```

Comment:
maybe it'd useful to say that if this parameter is not specified then the oldest project will be targeted

Reply:
Thank you for your feedback, I've created a ticket to address it in a separate PR https://hashicorp.atlassian.net/browse/HCPF-644. Since it is a large change, I will discuss it with the team and we will implement it separately.